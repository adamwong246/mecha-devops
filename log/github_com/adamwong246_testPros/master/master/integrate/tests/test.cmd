run,--rm=true,--cidfile=./log/github_com/adamwong246_testPros/master/master/integrate/tests/test.dpid,github_com-adamwong246_testpros/master:master,npm,run,test