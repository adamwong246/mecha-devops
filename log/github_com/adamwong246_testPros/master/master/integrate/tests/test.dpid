158d2de9c95cd62dd3385082cd7a0723c655f9d8b5bec54507c93201c008804c