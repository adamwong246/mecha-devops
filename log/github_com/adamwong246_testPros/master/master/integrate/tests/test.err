npm info it worked if it ends with ok
npm info using npm@3.10.10
npm info using node@v6.10.0
npm info lifecycle test-proj@1.0.0~pretest: test-proj@1.0.0
npm info lifecycle test-proj@1.0.0~test: test-proj@1.0.0
 PASS  __tests__/0.js
  âœ“ adds 1 + 2 to equal 3 (7ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        1.865s
Ran all test suites.
npm info lifecycle test-proj@1.0.0~posttest: test-proj@1.0.0
npm info ok 
