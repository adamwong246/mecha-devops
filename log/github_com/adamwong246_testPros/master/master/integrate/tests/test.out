
> test-proj@1.0.0 test /usr/src/app
> jest

